### PR TITLE
Added Envoy Operator and Solo.io

### PR DESCRIPTION
**Pull request by ilevine**

### CODE REVIEW

**Comment by caniszczyk** [comment]
@ilevine can you repush with 'git commit -s' (sign the commit per DCO)

**Comment by ilevine** [comment]
@caniszczyk  I noticed the deploy check failed, but I'm not sure it has to do with my change

**Comment by richarddli** [comment]
I kicked the deploy

**Comment by caniszczyk** [comment]
not sure why it fails but LGTM, will merge in

File: _data/home.yml [hunk 1]
```diff
@@ -135,11 +139,16 @@ projects:
    website: https://github.com/heptio/gimbal
    github: https://github.com/heptio/gimbal
    description: Heptio Gimbal is a layer 7 load balancing platform built on Kubernetes, the Envoy proxy, and Heptio Contour, a Kubernetes Ingress controller. It provides a scalable, multi-team, and API-driven ingress tier capable of routing Internet traffic to multiple upstream Kubernetes clusters and traditional infrastructure technologies such as OpenStack.
+ - name: Envoy Operator
+   logo: /img/logos/envoy-operator.svg
+   website: https://github.com/solo-io/envoy-operator
+   github: https://github.com/solo-io//envoy-operator
```
Comment (richarddli) [inline _data/home.yml]: I believe the two `//` here is what's breaking the parsing

**Pull request merged**